lib/processors/lib/utils.py

```diff
@@ -11,7 +12,11 @@ def get(self):
             userid = self.get_argument('userid')
             privatekey_pem = self.get_argument('privatekey', None)
             publickey_pem = self.get_argument('publickey', None)
-            user = User(userid, publickey_pem=publickey_pem, privatekey_pem=privatekey_pem)
+            user_blob = get_user(userid)
```

Comment:
This was the bug issue though that Peyman was talking about. Our User object now has user_name's inside of it. So when a handler gets called by the outside world, we need a way to create a user object that has the user_name. If you look at my mirror or truth processor, they both include the user's name (and need them).

So if we're not going to do this, how exactly does a user object that includes user name get passed to the handler?